api/defaults.go

```diff
@@ -285,7 +289,7 @@ func defaultNodeConfig(installationID string, request *requests.CreateAccount, o
 			URL:     request.UpstreamConfig,
 		}
 	} else {
-		nodeConfig.UpstreamConfig.URL = defaultNetworks[0].RPCURL
+		nodeConfig.UpstreamConfig.URL = mainnet(request.WalletSecretsConfig.StatusProxyStageName).RPCURL
```

Comment:
This is not using our proxy, should i set it to the proxy?

Reply:
I don't think so,  `RPCURL` and `FallbackURL` are used for direct access to the providers in case the proxy fails, we shouldn't touch those

Reply:
<img width="617" alt="image" src="https://github.com/user-attachments/assets/82f7da68-76aa-4980-ac2d-b8cf7c0c9ee7">

Reply:
sorry, wrote before checking what was actually being set here.
I need to check where this is used,  I thought we used the networks list everywhere